spec/unit/mutant/integration_spec.rb

```diff
@@ -10,6 +10,20 @@
     subject { object.setup }
     it_should_behave_like 'a command method'
   end
+
+  describe '.setup' do
+    subject { described_class.setup(kernel, name) }
+
+    let(:name)   { 'null'                                  }
+    let(:kernel) { instance_double(Kernel.singleton_class) }
```

Comment:
is `instance_double(Kernel.singleton_class)` functionally different from `class_double(Kernel)`?